cmd/openshift-install/create.go

```diff
@@ -488,6 +500,67 @@ func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset
 	return nil
 }
 
+// When bootstrap on SNO deployments, we should not remove the bootstrap node prematurely,
+// here we make sure that the deployment is stable.
+// Given the nature of single node we just need to make sure things such as etcd are in the proper state
+// before continuing.
+func waitForStableSNOBootstrap(ctx context.Context, config *rest.Config) error {
+	timeout := 5 * time.Minute
+
+	// If we're not in a single node deployment, bail early
+	if assetStore, err := assetstore.NewStore(command.RootOpts.Dir); err == nil {
+		if installConfig, err := assetStore.Load(&installconfig.InstallConfig{}); err == nil && installConfig != nil {
+			config := installConfig.(*installconfig.InstallConfig).Config
+			if machinePool := config.ControlPlane; machinePool != nil {
+				isSNO := *machinePool.Replicas == int64(1)
+				if !isSNO {
+					return nil
+				}
+			}
+		}
+	}
+
+	snoBootstrapContext, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+
+	untilTime := time.Now().Add(timeout)
+	timezone, _ := untilTime.Zone()
+	logrus.Warn("Detected Single Node Deployment")
```

Comment:
Why it should be warning? I assume it can be confusing for user to see warnings no?

Reply:
My thinking was that since we're adding a new wait time, it's worth bringing attention as to why they see a new 5min timer, but if you think it doesn't follow the flow, I can just make it a regular message

Reply:
Updated to be an info message, since you're right it doesn't really require their attention.